src/index.js

```diff
@@ -0,0 +1,113 @@
+const MAX_TIMEOUT_ATTEMPTS = 3;
+const MAX_TIMEOUT = 1000;
+
+
+function main(client) {
+	let timeoutAttempts = 0;
+	let appliedClasses = [];
+	let unappliedFunctions = new Set();
+	let appliedFunctions = new Set();
+
+	function retrieveEvolvCssAsset() {
+		const styleSheets = document.getElementsByTagName("link");
+
+		for (let i = 0; i < styleSheets.length; i++) {
+			const link = styleSheets[i];
+			if (link.href &&
+					link.href.indexOf('evolv.ai') >= 0 &&
+					link.href.indexOf('assets.css') >= 0) {
+				return link;
+			}
+		}
+
+		return undefined;
+	}
+
+	function retrieveEvolvJsAsset() {
+		const scripts = document.scripts;
+	
+		for (let i = 0; i < scripts.length; i++) {
+			const script = scripts[i];
+			if (script.src && script.src.indexOf('evolv.ai') >= 0 && script.src.indexOf('assets.js') >= 0) {
+				return script;
+			}
+		}
+	
+		return undefined;
```

Comment:
See comment above about using "evolv.ai" as a marker. And also shortening with this:
```suggestion
		return document.querySelector('link[href *= "evolv.ai"][href $= "assets.js"]');
```

Reply:
Im looking for the environment id now so localhost should work